Resources/ServerInfo/Guidebook/Antagonist/SpaceNinja.xml

```diff
@@ -7,32 +7,32 @@
 
   You are an elite mercenary that [color=#66FF00]The Spider Clan[/color] has sent to wreak all kinds of havoc on the station. You are equipped to keep it silent-but-deadly.
 
-  Whether you decide to mercilessly pick off the station's crew one by one or stay out of trouble, your discipline has taught you that [italic]your objectives must be at least attempted[/italic]. For honor!
+  Whether you get bloody or stay out of trouble, your discipline has taught you that [italic]your objectives must be at least attempted.[/italic] For honor!
 
   ## Standard Equipment
-  You begin implanted with a [color=cyan]death acidifier[/color], so if you are captured or KIA all your precious equipment is kept out of enemy hands.
+  You begin implanted with a [color=cyan]death acidifier,[/color] so if you are captured or KIA all your precious equipment is kept out of enemy hands.
 
-  Your bag is full of [bold]tools[/bold] and a [bold]survival box[/bold]. You also carry a [color=cyan]jetpack[/color] and a [color=cyan]pinpointer[/color] that will lead you to the station.
+  Your bag is full of [bold]tools[/bold] and a [bold]survival box.[/bold] You also carry a [color=cyan]jetpack[/color] and a [color=cyan]pinpointer[/color] that will lead you to the station.
 
   <Box>
     <GuideEntityEmbed Entity="JetpackBlack"/>
     <GuideEntityEmbed Entity="PinpointerStation"/>
   </Box>
   
-  ## Ninja Suit & Boots
+  ## Shinobi Shozoku
```

Comment:
I don't think everyone will know what this means or if it is even accurate. In-game it is called "space ninja suit", so keep it the old way.